firewall/context/suricata-installation.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -e
+source /etc/os-release
+
+if [ "${ID}" = "ubuntu" ] ; then
+    echo "Ubuntu - Install suricata from suricata ppa repository"
+    apt-get update --quiet
+    apt-get install --yes --no-install-recommends software-properties-common
+    add-apt-repository --yes ppa:oisf/suricata-stable
+    apt-get update --quiet
+    apt-get install --yes --no-install-recommends suricata
+else
+    echo "Debian - Install suricata from debian repository"
+    rm -f /etc/logrotate.d/suricata
+    apt-get update --quiet
+    apt-get install --yes --no-install-recommends suricata suricata-update
```

Comment:
We should install suricata from debian-testing, otherwise we'll get version 4.x.
see https://tracker.debian.org/news/1104221/suricata-1502-3-migrated-to-testing/

Reply:
OK, will do thanks for the hint